packages/cli/src/api/parsers/utils.js

```diff
@@ -112,7 +112,18 @@ function isPayloadValid(payload, options = {}) {
 function isTransifexCall(node) {
   const { callee } = node;
   if (!callee) return false;
-  if (_.includes(['t', 'useT', '$t'], callee.name)) { return true; }
+
+  let fnName;
+  if (callee.type === 'MemberExpression'
+    && callee.property.type === 'Identifier') {
+    fnName = callee.property.name;
+  } else if (callee.type !== 'Identifier') {
+    return false;
+  } else {
+    fnName = callee.name;
+  }
+
+  if (_.includes(['t', 'useT', '$t', 'translateLocale', 'ut'], fnName)) { return true; }
```

Comment:
Not sure if `translateLocale` will work here, as this function takes the locale as a first argument. I think the code using this function expects the actual string to be on the first argument, so this will probably not work.

For example `translateLocale('en', 'Hello')` will make the CLI detect `en` as the source string.

Also `useT` should be removed from CLI, since this has changed when we made `useT` return a function. The code here was used when `useT` returned a translated string, e.g. `useT('Hello')`. We should also remove the related tests.